framework/page_forms/spotify_home_form.py

```diff
@@ -0,0 +1,30 @@
+from forms.base_form import BaseForm
+from browser.py_quality_services import PyQualityServices
+from selenium.webdriver.common.by import By
+from resources.locator_constants import LocatorConstants
+
+
+class SpotifyHomeForm(BaseForm):
+    __form_name = "Popular artists"
+    __element = PyQualityServices.element_factory
+    __search_btn = __element.get_button((By.XPATH, ("//*[contains(@class, 'search')]")), "Search Button")
```

Comment:
you don't need to use words like "Button" in friendly names, as it will be added automatically because you are using get_button. You can just pass "Search"